sw/device/rom_exts/manifest.rs.tpl

```diff
@@ -0,0 +1,18 @@
+/// Copyright lowRISC contributors.
+/// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+/// SPDX-License-Identifier: Apache-2.0
+
+% for name, region in regions:
+/// Manifest field ${name}:
+/// Offset from the base, size in bytes and words.
```

Comment:
Unfortunately, you'll need to replicate this comment for each const.

Reply:
Each const needs its own doc-comment, or the documentation generator won't associate the comment with all the consts.

Reply:
Done.